components/resources/quizzes/careerQuiz/career-quiz.tsx

```diff
@@ -34,128 +37,105 @@ const CareerQuiz = () => {
   // create results state object that
   // create custom type -- based on schema type in database
   const [stage, setStage] = useState<Stage>(Stage.START);
-
+  const [open, setOpen] = useState(true);
```

Comment:
what does open mean in the context of this animation? what does it mean when this state is true vs false?

Reply:
this state variable toggles the animation when it is set to false. I've changed the name to triggerAnimation